rand_chacha/Cargo.toml

```diff
@@ -15,10 +15,11 @@ categories = ["algorithms", "no-std"]
 edition = "2018"
 
 [dependencies]
-rand_core = { path = "../rand_core", version = "0.6.0" }
+rand_core = { path = "../rand_core", version = "0.6.0", default-features = false }
```

Comment:
`rand_core` [doesn't have any default features](https://github.com/rust-random/rand/blob/master/rand_core/Cargo.toml#L18), so this change does nothing IIUC